apollo-gradle-plugin/src/test/groovy/com/apollographql/apollo/gradle/integration/BasicAndroidSpec.groovy

```diff
@@ -350,7 +350,7 @@ class BasicAndroidSpec extends Specification {
   }
 
   def cleanupSpec() {
-    FileUtils.deleteDirectory(testProjectDir)
+    //FileUtils.deleteDirectory(testProjectDir)
```

Comment:
Can this be a mistake?

Reply:
Definitely a mistake, I do this during debugging, I forgot to uncomment. Thanks for noticing !